Clappr/Classes/Base/Options.swift

```diff
@@ -5,9 +5,22 @@ public let kSourceUrl = "sourceUrl"
 public let kMediaControl = "mediaControl"
 public let kFullscreen = "fullscreen"
 public let kFullscreenDisabled = "fullscreenDisabled"
+public let kFullscreenByApp = "fullscreenByApp"
 public let kStartAt = "startAt"
 public let kAutoPlay = "autoplay"
 public let kPlaybackNotSupportedMessage = "playbackNotSupportedMessage"
 public let kMimeType = "mimeType"
 public let kDefaultSubtitle = "defaultSubtitle"
 public let kDefaultAudioSource = "defaultAudioSource"
+
+struct OptionsWrapper {
```

Comment:
Since seems to me that the functionality of this class is unbox the optional values could make more sense call it OptionsUnboxer?